torch/linalg/__init__.py

```diff
@@ -2134,6 +2134,9 @@
     It is possible to compute the solution of the system :math:`XA = B` by passing the inputs
     :attr:`A` and :attr:`B` transposed and transposing the output returned by this function.
```

Comment:
This comment is stale. We should remove it in a different PR. Also, the note that this function works with CSR tensors might be better placed in the paragraph where we discuss dtype and batched support above. Not for this PR tho.